### PR TITLE
Add discription of page_size=-1 in help doc

### PR DESCRIPTION
**Pull request by xiangge**
Add discription of page_size=-1 in help doc
JIRA: PDC-2181

### CODE REVIEW

File: pdc_client/runner.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def setup(self):
         self.parser.add_argument('--json', action='store_true',
                                  help='display output as JSON')
         self.parser.add_argument('--page-size', dest='page_size', type=int,
-                                 help='change page size in response')
+                                 help='change page size in response, a -1 means that get all pages of data in one request')
```
Comment (hluk) [inline pdc_client/runner.py]: `a -1 ...` -> `-1 ...` (remove `a`)

Comment (xiangge) [reply pdc_client/runner.py]: No sure if the sentence is okay, leave it to Chuang for having a look. @simozhan

Comment (simozhan) [reply pdc_client/runner.py]: I agree with @hluk, 'a' is unnecessary :)

Comment (xiangge) [reply pdc_client/runner.py]: @hluk and @simozhan changed :)

**Review by hluk — APPROVED** [review]

**Review by simozhan — APPROVED** [review]
LGTM

**Pull request merged**